pegasus/sites.v3/code.org/public/index.haml

```diff
@@ -42,31 +41,3 @@ social:
   = view :donor_slider
 
 = view :homepage_video
-
-
-:javascript
-
-  $(document).ready(function() {
-    // For IE8, can't start with opacity in the HTML, so we made things display: none.  Now we show them but
-    // set opacity to 0, ready for code to start showing them by fading them in.
-    $("#petitionexpand").css({display: "block", opacity: 0});
-  });
-
-  $(document).ready(function() {
-    $.ajax({
-      type: "GET",
-      url: '/dashboardapi/user_hero',
-      //data: "id=" + id, // appears as $_GET['id'] @ ur backend side
-      complete: function(data, status) {
-        if (data.responseText == "" || status !== "success") {
-          $("#petitionexpand").animate({opacity:1}, { duration: 2000 });
-          $('#user_hero').css({display: "none"});
-          //$("#petitionaction").animate({opacity:1}, { duration: 2000 });
-        } else {
-          $("#petitionexpand").css({display: "none"});
-          // data is ur summary
-          $('#user_hero').html(data.responseText, status);
-        }
-      }
-    });
-  });
```

Comment:
right, the `petitionexpand` div will be visible (but not expanded) on initial load, the contained `form` div will not be 'expanded' (via `expandPetition()`) until user action.